cloudformation/auxiliary_templates/custom_balancer/README.md

```diff
@@ -0,0 +1,59 @@
+# Deadline Custom Job Balancer
+
+This CloudFormation template deploys a sidecar Lambda function that controls the maxWorkerCount for each job within a Queue to implement functionality similar to the Deadline10 Balancer options.
+It implements a modified version of the Deadline 10 weighting algorithm: https://docs.thinkboxsoftware.com/products/deadline/10.4/1_User%20Manual/manual/job-scheduling.html#weighted-scheduling
+
+## Resources Created
+1. **Lambda Function**: An AWS Lambda function named `DeadlineWeightedJobBalancer` is created. This is the main driver which calculates and controls the `maxWorkerCount` setting for jobs.
+2. **EventBridge Schedule**: A schedule is created to trigger the Lambda function once every minute.
+3. **IAM Role**: An IAM role `BalancerSchedulerRole` is created and used by the Schedule to trigger the Lambda function.
+4. **SQS Queue**: An SQS Queue `deadline-unprocessed-balancing-events` is created to act as a DeadLetterQueue for the Schedule.
+5. **UAM Role**: An IAM role `DeadlineWeightedJobBalancerRole` is created to be used as the Lambda execution role for our `DeadlineWeightedJobBalancer`.
+
+## Prerequisites
+
+Before deploying this CloudFormation template, check you have the following as per your chosen method of notifications:
+
+1. The AWS CLI installed and configured with your AWS credentials (if setting up using CLI option).
```

Comment:
A second prerequisite:

2. A Deadline Cloud Farm and Queue deployed in your account.